deploy/olm-catalog/openshift-template-service-broker-manifests/4.4/openshifttemplateservicebrokeroperator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+---
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: openshifttemplateservicebrokeroperator.v4.4.0
+  namespace: placeholder
+  annotations:
+    alm-examples: '[{"apiVersion":"osb.openshift.io/v1","kind":"TemplateServiceBroker","metadata":{"name":"template-service-broker","namespace":"template-service-broker"},"spec":{}}]'
```

Comment:
Would be nice to prettify this:
```json
[
  {
    "apiVersion": "osb.openshift.io/v1",
    "kind": "TemplateServiceBroker",
    "metadata": {
      "name": "template-service-broker",
      "namespace": "template-service-broker"
    },
    "spec": {}
  }
]
```

Reply:
We'd have to use a multi-line yaml string since it's just a string, but that's probably fine

Reply:
While I tend to agree, the other 3 versions of the files (4.1, 4.2, 4.3) use a single line. And since this is going to be removed in 4.5, I would vote not to fix this.